### ISSUE TITLE
Evaluating multiple scores

### ISSUE THREAD

**Issue by MasterManggis** [manuel-calzolari/sklearn-genetic#31]
Is there any way to return multiple scores at the same time, and refer one metric as an estimator like GridSearchCV?

**Comment by manuel-calzolari** [comment]
At the moment only a single scorer can be provided, similarly to other feature selection methods available in scikit-learn, such as RFECV.
